src/jit/compiler.h

```diff
@@ -8843,6 +8843,8 @@ void * CompAllocator::ArrayAlloc(size_t elems, size_t elemSize)
 inline
 LclVarDsc::LclVarDsc(Compiler* comp)
     :
+#ifndef _TARGET_X86_
+    // TODO-x86 - Need initialization of the _lvArgReg and _lvOtherArgReg for x86?
```

Comment:
I can't speak for this specific issue, but IMO initializing to REG_STK implies that it is a valid value. I think they should really be initialized to REG_NA, so that we ensure that they are set to valid values during compilation, rather than relying on a default. For lvOtherArgReg, it should presumably be the "uninitialized/not applicable" value if there is only one register.